src/complex/Filter/Actions/ImportH5ObjectPathsAction.cpp

```diff
@@ -48,15 +48,14 @@ Result<> ImportH5ObjectPathsAction::apply(DataStructure& dataStructure, Mode mod
   bool preflighting = (mode == Mode::Preflight);
 
   H5::FileReader fileReader(m_H5FilePath);
-  H5::ErrorType errorCode;
-  const DREAM3D::FileData fileData = DREAM3D::ReadFile(fileReader, errorCode, preflighting);
-  if(errorCode < 0)
+  const Result<DREAM3D::FileData> fileData = DREAM3D::ReadFile(fileReader, preflighting);
```

Comment:
Only the `DataStructure` is needed for import.

Reply:
fixed by 8303467c41b4baab2ef4098e3bccebfaf4923ad9